bazel/emscripten_toolchain/toolchain.bzl

```diff
@@ -1018,6 +1018,11 @@ def _impl(ctx):
             flags = ["-Werror"],
             features = ["wasm_warnings_as_errors"],
         ),
+        flag_set(
+            actions = ALL_COMPILE_ACTIONS,
+            flags = ["-DSTANDALONE_WASM=1", "-DEMSCRIPTEN_STANDALONE_WASM=1"],
+            features = ["wasm_standalone"],
+        ),
```

Comment:
I wonder if project that want to build a special version of the code for standalone mode should just define this themselves?  This is what they would have to do in the non-bazel case where `-sSTANDALONE_WASM` is only a link-time flag.

Reply:
It guess it maybe makes more sense for bazel to have different builds for different configurations, but in general I would hope that folks who what to build wasm libraries would just build a single library that could be used in both standalone and non-standalone mode, and they would not need to know at compile time.

We already have PIC and non-PIC and threads and non-threads builds, I would not want to encourage yet another axis on which to vary libraries.. but I guess maybe bazel hides all this so it matters less?